examples/persistence-storage/.env

```diff
@@ -0,0 +1,8 @@
+DB_PORT=5432
+DB_USER=user
+DB_PASSWORD=s3cr3t
+DB_DATABASE_NAME=kamu
+
+KAMU_PORT=8080
+KAMU_AUTH_GITHUB_CLIENT_ID=361a3b4fda86d0234d2f
```

Comment:
GitHub values are not examples, don't store them in Git.

Reply:
Note that I did embed GH clientID/secret for "localhost" oauth app in [kamu-deploy-example](https://github.com/kamu-data/kamu-deploy-example/blob/master/environments/minikube/kamu-api-server/values.yaml.gotmpl#L8). It seemed low risk and the best way to make GH auth work out of the box for the example deployment.

Reply:
left it as it